### DISCUSSION TITLE
What is the different between IDialogService and IRegionDialogService?

### DISCUSSION THREAD

**Discussion by Chorusapp** [PrismLibrary/Prism#2533] [WPF]
Hi.

I watched Brian's course on Pluralsight, Introduction to Prism for WPF.
And, I am watching his stream on Youtube, Prism for WPF - Building Outlook nowadays.

He uses IDialogService interface on Pluralsight's course but, uses IRegionDialogService inteface on Youtube.

It is really confusing.

I I want to know what is the difference between them.

**Comment by brianlagunas** [comment]
The IDialogService interface is part of Prism and is used for showing basic dialogs.  The IRegionDialogService is a custom interface I wrote specifically for the app I was building in the YouTube video that handled region navigation within dialogs.

**Reply by Chorusapp** [reply]
Thanks to you, I got to understand.
